consensus/broadcast/shardChainMessenger.go

```diff
@@ -196,6 +196,7 @@ func (scm *shardChainMessenger) PrepareBroadcastHeaderValidator(
 	err = scm.delayedBlockBroadcaster.SetHeaderForValidator(vData)
 	if err != nil {
 		log.Error("shardChainMessenger.PrepareBroadcastHeaderValidator", "error", err)
+		return
```

Comment:
this return is redundant

Reply:
I would keep it just for the case when the method would be extended by someone who would forgot to added back